### PR TITLE
Various changes and fixes for missed broken functionalities

### PR DESCRIPTION
**Pull request by Ilithor**

### CODE REVIEW

File: server/handlers/user.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import _ from 'lodash';
-import bcrypt from 'bcrypt';
+//import bcrypt from 'bcrypt';
```
Comment (jeffersoneagley) [inline server/handlers/user.js]: Delet dis?

Comment (Ilithor) [reply server/handlers/user.js]: Deletted

File: server/handlers/user.js [hunk 2]
```diff
@@ -13,7 +12,7 @@ import { dataUri } from '../util/multer';
 import {
   findById,
   findUserAndUpdateImage,
-  findAndUpdatePassword,
+  //findAndUpdatePassword,
```
Comment (jeffersoneagley) [inline server/handlers/user.js]: derp

File: src/components/login/loginContext.js [hunk 3]
```diff
@@ -1,9 +1,12 @@
 import React, { createContext, useContext, useState } from 'react';
 import axios from 'axios';
-import bcrypt from 'bcryptjs';
+//import bcrypt from 'bcryptjs';
```
Comment (jeffersoneagley) [inline src/components/login/loginContext.js]: derp

**Pull request merged**